lib/elixir/src/elixir_dispatch.erl

```diff
@@ -329,11 +329,24 @@ elixir_imported_macros() ->
 
 %% Inline common cases.
 check_deprecation(Meta, ?kernel, to_char_list, 1, E) ->
-  elixir_errors:warn(?line(Meta), ?key(E, file), "Use Kernel.to_charlist/1 instead");
+  Message = "Kernel.to_char_list/1 is deprecated. Use Kernel.to_charlist/1 instead",
+  elixir_errors:warn(?line(Meta), ?key(E, file), Message);
 check_deprecation(_, ?kernel, _, _, _) ->
   ok;
 check_deprecation(_, erlang, _, _, _) ->
   ok;
+check_deprecation(Meta, 'Elixir.System', stacktrace, 0, #{contextual_vars := Vars} = E) ->
```

Comment:
I just realised - should we also warn for `:erlang.get_stacktrace`? Otherwise we suppress the Erlang's warning completely.

Reply:
Erlang get_stacktrace is getting its own warning, we are not touching it. The suppression is only on the v1.6 branch.